### ISSUE TITLE
Docker Swarm!! & ENV values

### ISSUE THREAD

**Issue by abhranil26** [almarklein/mypaas#41]
Absolutely love the minimalistic approach, tried it out works Traefik+ My-Paas works great and fast.
Just wondering if Docker Swarm will be added in the future.
Also would love if env values could be updated from passing a command instead of deploying repeatedly.
Could be a perfect caprover alternative then.

**Comment by almarklein** [comment]
> Just wondering if Docker Swarm will be added in the future.

I don't think so. MyPaas is based on Traefik. Supporting other "backends" would add an order of magnitude in complexity.

**Comment by almarklein** [comment]
> Also would love if env values could be updated from passing a command instead of deploying repeatedly.

That seems reasonable.
